store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/operation/handler/impl/FederatedOperationHandler.java

```diff
@@ -105,7 +108,8 @@ private Object mergeResults(final Iterable resultsFromAllGraphs, final Federated
 
             return rtn;
         } catch (final Exception e) {
-            throw new OperationException(String.format("Error while merging results. %s", Objects.toString(e.getMessage(), "")), e);
+            final Object graphs = context.getVariable(MERGE_FUNCTION_GET_GRAPHS);
+            throw new OperationException(String.format("Error while merging results from graphs: %s due to: %s", graphs.toString(), Objects.toString(e.getMessage(), "")), e);
```

Comment:
Yeah, I have no memory of using it, but it was there before. seems maybe better that having a "ternary" or "if null"
🤷‍♂️

Reply:
I don't think this is required at all. `String.format` is null safe. If `e.getMessage()` is `null`, it will automatically replace it with empty string, we don't need to do this.

Reply:
I guess in a situation when the object "e" could throw a null pointer this is safe. but in a Catch e will never be null. 😆

Reply:
I think it's for tidyness. Some exceptions might not have message like NullPointer.

Reply:
done